src/node/node_state.h

```diff
@@ -1291,6 +1295,12 @@ namespace ccf
       }
 
       consensus->periodic(elapsed);
+
+      // if (sm.check(State::partOfNetwork))
+      // {
+      //   const auto tx_id = consensus->get_committed_txid();
+      //   indexer->update_strategies(elapsed, {tx_id.first, tx_id.second});
+      // }
```

Comment:
TODO: Revert this before merging. Temporarily disabling the indexing to confirm this removes the perf drop, since I can't reproduce it locally. Of course this will break a bunch of tests, so maybe we won't get any perf results from it...